src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -844,4 +832,14 @@ export default {
 		height: $clickable-area;
 	}
 }
+
+::v-deep button.action-item,
+::v-deep .action-item__menutoggle {
+	// Fix screensharing icon width
```

Comment:
is this an upstream issue to fix separately ?

on the other hand, once we switch to material design icons here this might disappear.

note that I had a quick look back then and I'm not sure we'll find icons that match all these menu actions

Reply:
Once we'll the button component ready, covering all use cases, and actions will use it, we will be able to get rid of this I think